include/mfmg/dealii/amge_host.templates.hpp

```diff
@@ -233,6 +241,10 @@ AMGe_host<dim, MeshEvaluator, VectorType>::compute_local_eigenvectors(
       for (unsigned int j = 0; j < n_dofs_agglomerate; ++j)
         eigenvectors[i][j] = lapack_eigenvectors[j][i];
   }
+  else
+  {
+    ASSERT(true, "Unknown eigensolver type");
+  }
```

Comment:
@Rombur was there a rational for not having a `else`-clause?

Reply:
Not really.

Reply:
Doesn't mean you shouldn't do that.  Just putting it here for the record.